package.json

```diff
@@ -72,15 +78,10 @@
     "tailwindcss": "^3.4.3",
     "typescript": "^5.4.5",
     "unist-util-visit": "^5.0.0",
-    "uuid": "^10.0.0",
-    "@types/diff": "^5.2.2",
-    "@types/lodash.merge": "4.6.9",
-    "@parcel/watcher": "^2.4.1",
-    "commander": "^12.1.0",
-    "concurrently": "^8.2.2",
-    "cross-env": "^7.0.3"
+    "uuid": "^10.0.0"
   },
   "devDependencies": {
+    "@changesets/cli": "^2.27.9",
```

Comment:
Is this package.json file changes required? Any ideas?

Reply:
This changes on package.json is only needed to make docker github workflow pass...